Documentation/tutorial/intro_toc.md

```diff
@@ -64,6 +64,7 @@ Property Name | Type              | Description
 ------------- | ----------------- | ---------------------------
 *name*        | string            | Specifies the title of the *TOC Item*.
 *href*        | string            | Specifies the hyperlink of the *TOC Item*.
+*tocHref*     | string            | Specified another TOC file to be expanded in the currrent *TOC Item*.
 *homepage*    | string            | Specifies the homepage of the *TOC Item*. It is useful when *href* is linking to a folder.
```

Comment:
reserve for use, but move to advanced section.

---
In reply to: [72391681](https://github.com/dotnet/docfx/pull/537#discussion_r72391681) [](ancestors = 72391681)